scalameta/io/shared/src/main/scala/scala/meta/io/AbsolutePath.scala

```diff
@@ -16,7 +16,11 @@ import scala.meta.internal.io.{FileIO, PathIO}
   @deprecated("Use toString() instead", "1.8")
   def absolute: String = toString()
 
-  def toRelative: RelativePath = toRelative(PathIO.workingDirectory)
+  // NOTE: Anticipating exceptions in this method is just plain terrible.
+  // It's not exceptional at all that this AbsolutePath is not relative to the
+  // working directory. Fix this once https://github.com/scalameta/scalameta/issues/821
+  // is settled.
+  def toRelative: Option[RelativePath] = scala.util.Try(toRelative(PathIO.workingDirectory)).toOption
```

Comment:
I suggest we remove this method altogether until we work out #821. Since that issue has been brought up multiple times over the last couple weeks, I also suggest we bump its priority to critical, lower than only the issues that block def macros.